pkgs/by-name/ni/nixos-firewall-tool/nixos-firewall-tool-nftables.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+show_help() {
+    echo "nixos-firewall-tool"
+    echo ""
+    echo "Can temporarily manipulate the NixOS firewall"
+    echo ""
+    echo "Open TCP port:"
+    echo " nixos-firewall-tool open tcp 8888"
+    echo ""
+    echo "Show all firewall rules:"
+    echo " nixos-firewall-tool show"
+    echo ""
+    echo "Open UDP port:"
+    echo " nixos-firewall-tool open udp 51820"
+    echo ""
+    echo "Reset firewall configuration to system settings:"
+    echo " nixos-firewall-tool reset"
```

Comment:
This can be combined into one echo with mutliline strings. Also echo already prints an empty line.

Reply:
This was copied verbatim from pkgs/by-name/ni/nixos-firewall-tool/nixos-firewall-tool.sh, and I've changed approach to detect for iptables/nftables support instead of having two separate files.